flags_test.go

```diff
@@ -36,3 +36,12 @@ func TestBackupProfileName(t *testing.T) {
 	assert.Equal(t, flags.name, "profile1")
 	assert.Equal(t, flags.resticArgs, []string{"backup", "-v"})
 }
+
+func TestBackupProfileNameAsRun(t *testing.T) {
+	_, flags, err := loadFlags([]string{"-r", "prof.ile1.backup", "--verbose", "--", "-v"})
+	require.NoError(t, err)
+	assert.False(t, flags.help)
+	assert.Equal(t, flags.name, "prof.ile1")
+	assert.True(t, flags.verbose)
+	assert.Equal(t, flags.resticArgs, []string{"backup", "-v"})
+}
```

Comment:
I would probably add another test with these arguments `[]string{"-r", "prof.ile1.backup", "--", "-v"}` and make sure the verbose flag is **not** set, so we can be sure it's not picked up from the `-v` we send directly to restic

and maybe another test to make sure a simple `[]string{"-r", "profile1.backup"}` is well understood

(you can never run enough tests 😆 )

Reply:
I did, though the Quality Gate seems to think they are too similar to write twice :grimacing: